static/includes/apps/AppsInstallWizardDefaultPorts.md

```diff
@@ -0,0 +1,5 @@
+&NewLine;
+
+All TrueNAS apps are assigned default port numbers.
+Accept the default port numbers, but if changing port number assignments, enter a number within the range 9000-65535.
```

Comment:
There is no longer a limitation on ports below 9000.
User can effectively assign 1-65535 (as long as its not already used).

0-1024 might require the application to have elevated privileges.